### ISSUE TITLE
Why search for a subtitle if there is a subtitle file next to the video with the same name?

### ISSUE THREAD

**Issue by peno64** [denimnani/service.subloader#8]
As the title says. Why does the service searches for a subtitle if there is a subtitle file next to the video with the same name?

**Comment by denimnani** [comment]
Hello there, yes it does. Go to settings then exceptions, then check for subtitles and the desired language. After this the service will stop if the subtitle is already loaded. Let me know if is working for you.

**Comment by peno64** [comment]
This is working if the subtitle is embedded in the video file like in mkv files but not is there is a .srt file next to the video file with the same name but extension .srt instead of .avi or mp4. How would the service also know in which language this .srt file?

**Comment by denimnani** [comment]
Yeah I see the issue, this way the service can't see the actually language for the subtitle. Try to rename the sub. Let's imagine that our language is English, then rename your subutitle to "YOUR SUBTITLE NAME"."en".srt the "en" is for English, you need to place there the iso code 639-1 for your actual language. Let me know if is working like this. Either way I will try to fix this issue natively.

**Comment by peno64** [comment]
Yes, adding the subtitle language to the srt file fixes this. It would be great if it would also work if the same of the subtitle file is the same as the video file (except extension). In that case it should not look at the language code.

Thank you
